### ISSUE TITLE
Region Flags are broken

### ISSUE THREAD

**Issue by TopGunz9** [Trojaner25/Rocket-Regions#100]
I have tried to use the flags on a region but a lot of them are broken. The region member and owner commands are broken as well. What I mean by this is that when I used the command it said that the flag was set to true, but the flag didnt actually change anything. So when I did /rflag Safezone godmode true, it didn't actually give people godmode in the region. Here are a list of the flags I have found that don't work.
-Godmode
-NoDestroy
-NoZombies (It's not really broken but it is glitched. The zombies still spawn but die instantly.
-NoEquipWeapon (It works fine for normal players, but it still applies to people even when they are a region member or owner)

**Comment by Prefektt** [comment]
Yeah, I also tried to use regions yesterday, and it was causing a lot of errors with my server. I'm not sure what's going on, but I had to remove zombies for the time being, since I can't have them in my RP town.

**Comment by Prefektt** [comment]
Errors like the ones mentioned above by @TopGunz9.

**Comment by bbgtired** [comment]
Yes, the NoZombies doesn't work anymore either, I'm not sure what to do with my RP server lol
